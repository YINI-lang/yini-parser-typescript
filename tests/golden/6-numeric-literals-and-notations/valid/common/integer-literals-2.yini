// Common edge-case decimal integers.
^ Integers
int1 = 0                     // zero
int2 = +0                    // zero
int3 = 1                     // smallest positive integer
int4 = -1                    // smallest negative integer
int5 = 9223372036854775807   // max 64-bit signed integer
int6 = -9223372036854775808  // min 64-bit signed integer
