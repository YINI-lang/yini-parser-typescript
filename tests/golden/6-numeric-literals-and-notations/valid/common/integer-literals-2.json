{
    "Integers": {
        "int1": 0,
        "int2": 0,
        "int3": 1,
        "int4": -1,
        "int5": 9223372036854775807,
        "int6": -9223372036854775808
    }
}
